README.md

```diff
@@ -21,8 +21,8 @@ to a rich text format, so make sure you are saving plain text.
 
 ## Install software dependencies
 
-At this itme, TolaActivity requires both Python 2 and Python 3. It has been thoroughly tested with versions
-2.7 and 3.6, and lightly tested with version 3.8.  Python 2 is necessary satisfy a dependency in a node module.
+At this itme, TolaActivity requires Python 3. It has been thoroughly tested with versions
+3.6, and lightly tested with version 3.8.
```

Comment:
We might want to say that devs should use Python 3.8 or greater.  I have do doubt that one of us will use the walrus operator soon, if we haven't already.

Reply:
I added that comment suggesting devs to use Python 3.8 or greater.